core/server/master/src/test/java/alluxio/master/file/FileSystemMasterTest.java

```diff
@@ -658,10 +663,16 @@ public void listStatusWithLoadMetadataNever() throws Exception {
 
     // getFileId should load metadata automatically.
     AlluxioURI uri = new AlluxioURI("/mnt/local/dir1");
-    List<FileInfo> fileInfoList = mFileSystemMaster.listStatus(uri,
-        ListStatusOptions.defaults().setLoadMetadataType(LoadMetadataType.Never));
-    Assert.assertEquals(0, fileInfoList.size());
-    Assert.assertEquals(4, mFileSystemMaster.getNumberOfPaths());
+    try {
+      mFileSystemMaster.listStatus(uri,
+          ListStatusOptions.defaults().setLoadMetadataType(LoadMetadataType.Never));
+      Assert.fail("listStatus() with load metadata NEVER should throw exception.");
```

Comment:
nit: I would simply say "exception expected" to avoid hardcoding class and enum name into the error message